addons/html_editor/static/tests/columnize.test.js

```diff
@@ -31,7 +31,7 @@ describe("2 columns", () => {
             contentAfterEdit:
                 columnsContainer(
                     column(6, `<p placeholder="Empty column" class="o-we-hint">[]<br></p>`) +
-                    column(6, `<p placeholder="Empty column" class="o-we-hint"><br></p>`)
+                    column(6, `<p><br></p>`)
```

Comment:
You should rename this test into `should display hint for focused empty column`.